### ISSUE TITLE
Accessing nested configuration serviceInfos

### ISSUE THREAD

**Issue by rpost** [avast/gradle-docker-compose-plugin#115]
Having nested config like this:

```
dockerCompose {
    myNested {
       // ...
    }
}
```

how do I access serviceInfos for `myNested`? The only solution I found (reading `ComposeExtension.methodMissing`) was: 
```
dockerCompose.myNested("" /* dummy argument here */).servicesInfos
```
which is somewhat akward.

**Comment by augi** [comment]
Hi, thanks for the report! I can confirm that you can access the nested settings using the syntax you provided, so call the method with one (or more) parameters (that are never used).

The improvement is under review now and so could be released in few hours.

Then it will be possible to use just plain `dockerCompose.myNested.servicesInfos`

**Comment by augi** [comment]
Released as `0.6.1`.

**Comment by rpost** [comment]
Thx!
